taf/src/test/java/google/framework/pages/GoogleCloudMainPage.java

```diff
@@ -0,0 +1,58 @@
+package google.framework.pages;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+
+public class GoogleCloudMainPage extends AbstractPage {
+    private static final String URL = "https://cloud.google.com/";
+    private static final String SEARCH_CALC = "Google Cloud Platform Pricing Calculator";
+
+    public GoogleCloudMainPage(WebDriver webDriver) {
+        super(webDriver);
+        PageFactory.initElements(webDriver, this);
+    }
+
+    @Override
+    public GoogleCloudMainPage openPage() {
+        webDriver.navigate().to(URL);
+        return this;
+    }
+
+    public GoogleCloudMainPage inputSearchAndSubmit() {
+        WebElement openLangs = waitWebElementAndClick("(//*[@class='devsite-select-toggle'])[1]");
+        WebElement choseLang = waitWebElementAndClick("(//li[text()='English'])[1]");
```

Comment:
WebElement openLangs 
WebElement choseLang 
and etc.
Why did you create this variable? These variables are never used. Check this through framework.
![ice_screenshot_20200210-125246](https://user-images.githubusercontent.com/37652976/74139346-46d27800-4c04-11ea-9d95-bfdc5a54c2b3.png)
Gray color in IDE says about this